File: Python/백준/Bronze/27326. 二人三脚 （Three－Legged Race）/二人三脚 （Three－Legged Race）.py
a = int(input())
aa = list(map(int,input().split()))
bb = {}
for i in aa:
    if i in bb.keys():
        bb[i]+=1
    else:
        bb[i]=1
for i in bb.keys():
    if bb[i]==1:
        print(i)

File: C#/백준/Bronze/27326. 二人三脚 （Three－Legged Race）/二人三脚 （Three－Legged Race）.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int[] x = new int[a];
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            for (int i=0;i<aa.Length;i++){
                x[aa[i]-1]++;
            }
            for (int i=0;i<a;i++){
                if(x[i]==1){
                    Console.WriteLine(i+1);
                    break;
                }
            }
        }
    }
}